src/app/header/menu/menu.component.scss

```diff
@@ -5,6 +5,8 @@ $menu-dropdown-border-color: rgba($color-accent, .6);
 $menu-dropdown-box-shadow-color: rgba(0, 0, 0, .4);
 $menu-dropdown-before-border-color: rgba($color-accent, .6);
 $menu-dropdown-sectioned-webkit-scrollbar-track-box-shadow-color: rgba(0, 0, 0, .5);
+$menu-dropdown-width: 200px;
+$menu-dropdown-sectioned-width: calc(#{$menu-dropdown-width} * 2);
```

Comment:
Unnecessary calc, this breaks positioning on ie because calc inside calc.  This works fine too: `$menu-dropdown-width * 2` 

Sass can calculate basic math without calc, calc should be used to calculate widths when relative widths are needed (ie. with percentages).